src/main.rs

```diff
@@ -80,7 +80,7 @@ arg_enum! {
     #[allow(non_camel_case_types)]
     pub enum OutputFormat {
         flamegraph,
-        flamegraph_stack_lines,
+        collapsed,
```

Comment:
It's due to the lack of description for each format that I'm more inclined to avoid including the word `flamegraph` so users will not think by mistake that this option generates a flamegraph as well.

how does `flamegraph_raw_collapsed` sound ?

Reply:
It sounds difficult to type. :)  Let's keep `collapsed` and see how it goes. If people find it confusing, we can add an alias or slowly deprecate the old name.